src/web-server/routes.js

```diff
@@ -9,33 +41,56 @@ export default (workbox, webServer) => {
         workbox.strategies.staleWhileRevalidate()
     );
 
+    workbox.routing.registerRoute(
+        wwwJsonRegex,
+        async ({ url }) => {
+            const path = url.pathname.match(wwwJsonRegex)[1];
+            // optionally check if we want to get file content,too
+            const getFileContent = !!url.searchParams.get('content');
```

Comment:
Let's not do this.  It doesn't really make sense, and you can just do two `fetch()`es to get the a) metadata; and b) contents if you need it.